openseeds/ols-2/speaker-guide.md

```diff
@@ -22,8 +22,7 @@ This program is led by the OLS team members, i.e. us - Bérénice Batut,
 Malvika Sharan and Yo Yehudi. You can reach out to us by emailing
 [{{ site.email }}](mailto:{{ site.email }}).
 
-We are currently running our [second cohort with 33
-projects](projects-participants.md)
+We are currently running our [second cohort with 33 projects](projects-participants.md)
```

Comment:
```suggestion
We are currently running our [second cohort with 33 projects]({% link openseeds/ols-2/projects-participants.md %})
```

Reply:
Thanks a lot! With this, all errors have been reconciled. 💃🏽

Currently, I'm adding Jekyll's _redirect_from_ to all the pages that were changed in the PR.